backend/de.metas.async/src/main/java/de/metas/async/processor/impl/WorkpackageProcessorTask.java

```diff
@@ -555,7 +555,7 @@ private void markSkipped(
 		final String msg = StringUtils.formatMessage("Skipped while processing workpackage by processor {}; workpackage={}", processorName, workPackage);
 
 		// log error (for later audit), but note that this exception can actually be quite frequent. so in order to not clutter the log, just log it on debug level
-		logger.debug(msg, skipException);
+		logger.error(msg, skipException);
```

Comment:
check my comment please :-)
if the problem is now so infrequest that we don't need the "debug" anymore, please adapt the comment accordingly